CONTRIBUTING.md

```diff
@@ -1,23 +1,21 @@
 # How to contribute
 
-In this repo we follow the conventions defined in Vincent Driessen's **[git branching model](http://nvie.com/posts/a-successful-git-branching-model/)**. See **[git-flow](https://github.com/nvie/gitflow)** if you want some git extensions to help you with the mentioned model.
-
-We **try** to use [Semantic Versioning](http://semver.org) and **Scrum** in our software development processes _as much as possible_. The [issues](https://github.com/iporaitech/pwr2-docker/issues) are grouped into [milestones](https://github.com/iporaitech/pwr2-docker/milestones) and most of the times will be labeled as: **Feature**, **Bug**, **Task**.  
+We **try** to use [Semantic Versioning](http://semver.org) and some sort of **Kanban** in our software development processes _as much as possible_. The [issues](https://github.com/iporaitech/pwr2-docker/issues) are grouped into [milestones](https://github.com/iporaitech/pwr2-docker/milestones) and most of the times will be labeled as: **Feature**, **Bug**, **Task**.
 
 A milestone can be either a **Release** or a **Hotfix**. A **Release** must increment the MINOR version number and usually corresponds to the work (issues) done in 1 sprint. A **Hotfix** must increment the PATCH version number and usually contains **Bug** fixes. Name milestones prefixing with the work **Release** or **Hotfix** accordingly.
```

Comment:
Typo detected: "... prefixing with the work".  Change the sentence to start with "We name milestones prefixing the name with the word ... "